Dockerfile

```diff
@@ -0,0 +1,19 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
+RUN npm install --silent
+RUN npm install react-scripts@3.4.0 -g --silent
```

Comment:
This should not be neccessary. Doesn't the project already use react-scripts/create-react-app?